great_expectations/analytics/config.py

```diff
@@ -45,6 +45,7 @@ class Config(GenericModel):
     data_context_id: Optional[UUID] = None
     oss_id: Optional[UUID] = None
     cloud_mode: bool = False
+    service: str = "gx-core"
```

Comment:
I'm not sure we need this; is it consumed somewhere? We do send this service value with every event we send to posthog. That is configured in [base event class here.](https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/analytics/base_event.py#L75)

Reply:
Thanks, I missed that!